R/derive_params_growth_age.R

```diff
@@ -183,25 +177,22 @@
 #'
 #' derive_params_growth_age(
 #'   advs,
-#'   by_vars = exprs(STUDYID, USUBJID, VISIT),
 #'   sex = SEX,
 #'   age = AGECUR,
 #'   age_unit = AGECURU,
 #'   meta_criteria = criteria,
-#'   parameter = VSTESTCD == "WEIGHT",
+#'   parameter = VSTESTCD == "HEIGHT",
 #'   analysis_var = VSSTRESN,
-#'   who_correction = TRUE,
 #'   set_values_to_sds = exprs(
-#'     PARAMCD = "WGASDS",
+#'     PARAMCD = "HTSDS",
 #'     PARAM = "Weight-for-age z-score"
 #'   ),
 #'   set_values_to_pctl = exprs(
-#'     PARAMCD = "WGAPCTL",
+#'     PARAMCD = "HTPCTL",
 #'     PARAM = "Weight-for-age percentile"
```

Comment:
Using `VSTESTCD == "HEIGHT"`, should we not have PARAM = "**Height**-for-age percentile" ?